net/aircrack-ng/Makefile

```diff
@@ -36,6 +38,19 @@ define Package/aircrack-ng/description
   WLAN tools for breaking 802.11 WEP/WPA keys
 endef
 
+define Package/airmon-ng
+  SECTION:=net
+  CATEGORY:=Network
+  DEPENDS:=+wireless-tools +ethtool +procps +CONFIG_PCI_SUPPORT:pciutils +CONFIG_USB_SUPPORT:usbutils
```

Comment:
The idea is to do the split first.  If it's working before it should keep that way after.

1.2-rc4 is nearly out (https://github.com/aircrack-ng/aircrack-ng/commit/e6893de1b907b1657d48fce70f37c08eadb842a2) without official announcement and tarball for download and I think it's still broken.

Personally I would keep away from 1.2-rcX for that single commit of 10000+ lines and the bug found in wpapsk.c